sqlpage/templates/shell.handlebars

```diff
@@ -10,17 +10,56 @@
         <link rel="manifest" href="{{manifest}}">
     {{/if}}
     <link rel="stylesheet" href="{{static_path 'sqlpage.css'}}">
-   {{#each (to_array css)}}
+    {{#each (to_array css)}}
         {{#if this}}
             <link rel="stylesheet" href="{{this}}">
         {{/if}}
-   {{/each}}
+    {{/each}}
+
+    {{!-- Ensure font is an array --}}
+    {{#if font}}
+        {{#if (not (isArray font))}}
+            {{!-- Wrap single font string in an array --}}
+            {{#set "font" (array font)}}
+        {{/if}}
+    {{/if}}
```

Comment:
You don't need that, we already have [to_array](https://sql.ophir.dev/custom_components.sql)